bindings/python/iota_sdk/wallet/sync_options.py

```diff
@@ -34,22 +45,52 @@ def __init__(self,
 
 
 class NftSyncOptions():
-    """Sync options for addresses from NFT outputs
+    """Sync options for addresses from NFT outputs.
+
+    Attributes:
+        basic_outputs (bool, optional): whether to sync basic outputs
+        nft_outputs (bool, optional): whether to sync NFT outputs
+        alias_outputs (bool, optional): whether to sync alias outputs
     """
 
     def __init__(self,
                  basic_outputs: Optional[bool] = None,
                  nft_outputs: Optional[bool] = None,
                  alias_outputs: Optional[bool] = None):
-        """Initialize NftSyncOptions
+        """Initialize NftSyncOptions.
         """
         self.basicOutputs = basic_outputs
         self.nftOutputs = nft_outputs
         self.aliasOutputs = alias_outputs
 
 
 class SyncOptions():
-    """The synchronization options
+    """The synchronization options.
+
+    **Attributes**
+    --------------
+    addresses : List[str], optional)
+        list of addresses to sync
+    address_start_index : int, optional
+        starting index for addresses
+    address_start_index_internal : int, optional
+        starting index for internal addresses
+    force_syncing : bool, optional
+        whether to force syncing
+    sync_incoming_transactions : bool, optional
+        whether to sync incoming transactions
+    sync_pending_transactions : bool, optional
+        whether to sync pending transactions
+    account : AccountSyncOptions, optional
+        account sync options
+    alias : AliasSyncOptions, optional
+        alias sync options
+    nft : NftSyncOptions, optional
+        NFT sync options
+    sync_only_most_basic_outputs : bool, optional
+        whether to sync only most basic outputs
+    sync_native_token_foundries : bool, optional
+        whether to sync native token foundries
```

Comment:
Some of them don't really look helpful, maybe copy the comments from the Rust code?